static/openapi/hss-spec.json

```diff
@@ -0,0 +1,3701 @@
+                                "remove_punctuation": false,
+                                "replace_unicode_quotes": true
+                            },
+                            "extract": {
+                                "enable": false,
+                                "logic": "function process(data) { return data; }"
+                            },
+                            "chunk": {
+                                "enable": false
+                            }
+                        }
+                    },
+                    "facet_fields": {
+                        "anyOf": [
+                            {
+                                "items": {
+                                    "type": "string"
+                                },
+                                "type": "array"
+                            },
+                            {
+                                "type": "null"
+                            }
+                        ],
+                        "title": "Facet Fields",
+                        "description": "Facet fields are attributes or properties of documents in a dataset that can be used to categorize and filter search results. List of fields in the document applicable for filtering results.",
+                        "default": [],
+                        "examples": [
+                            [
+                                "doc_name",
+                                "content"
+                            ]
+                        ]
+                    },
+                    "sort_fields": {
+                        "anyOf": [
+                            {
+                                "additionalProperties": {
+                                    "$ref": "#/components/schemas/SortOrderType"
+                                },
+                                "type": "object"
+                            },
+                            {
+                                "type": "null"
+                            }
+                        ],
+                        "title": "Sort Fields",
+                        "description": "List of fields in the document applicable for sorting results and direction. \n        Should be provided in `{ field_name: asc|desc }` format.  \n        **Note: Setting index level sorting affects\n        ranking and similarity score based ordering of search results.**",
+                        "default": {},
+                        "example": {
+                            "created_date": "desc",
+                            "doc_name": "asc"
+                        }
+                    },
+                    "index_job_interval": {
+                        "type": "integer",
+                        "minimum": 0.0,
+                        "title": "Index Job Interval",
+                        "description": "Set the time interval, in seconds, between each indexing job. Use 0 to index just once.",
```

Comment:
This doesn't make sense. It's a time interval, so how would zero make it index only one time?